src/components/Landing/QAs.tsx

```diff
@@ -1,15 +1,16 @@
+/* eslint-disable react/jsx-key */
 import { t, Trans } from '@lingui/macro'
 import ExternalLink from 'components/shared/ExternalLink'
-import { PropsWithChildren } from 'react'
+import { ReactNode } from 'react'
 import { Link } from 'react-router-dom'
 
-export const OverflowVideoLink = ({ children }: PropsWithChildren<{}>) => (
+export const OverflowVideoLink = ({ children }: { children: ReactNode }) => (
```

Comment:
what do we get with this approach vs `PropsWithChildren`, or is just that PropsWithChild<{}> is antipattern?

Reply:
more I think about it, more I don't like PropsWithChildren<{}>

Reply:
Yess!!! Great catch. 

`PropsWithChildren<{}>` is now an eslint error so that's why it's removed, but you're totally right, we can do React.FC 🚀

Reply:
done!

Reply:
YEAH